### ISSUE TITLE
A11Y - No voice over for bottom sheet credit card information

### ISSUE THREAD

**Issue by nbhasin2** [mozilla-mobile/firefox-ios#15318]
## Steps to reproduce
* Foxfooding Bug: 6/30/2023 15:25:14
* No particular steps
* Foxfooding Notes
* The cards information section of the drawer is not announced with VoiceOver (it receives focus but the container that was used for grouping text and image views is likely missing "isAccessibilityEnabled"/"isAccessibleElement" or similar). 

### Expected behavior
* It is expected to be announced on VoiceOver focus as "{Provider label from logo}, {Card Label}, {Provider * * * * last-four-digits}, Expires {MM}/{YY}", i.e. "Visa, Test Card, VISA ****4242, Expires 12/23

### Actual behavior
* No voice over for bottom sheet credit card information

### Device & build information
* Device: iPhone
* Build version: 32291
* First seen version: ? 
### Notes
Attachments: 
![](https://github.com/mozilla-mobile/firefox-ios/assets/8919439/d3ca1af2-ff75-46da-accd-7d33fdb4a361)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6878)

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Looks like the string and screen were changed, is this the fix? As the Save your card information securely with Firefox text is no longer displayed
Stefan Vladut Nishant Bhasin 

Please note that now the text on the screen is read correctly with voice over.

!image-20231222-131504.png|width=33.333333333333336%!

!image0 (4).png|width=33.333333333333336%!

**Comment by data-sync-user** [comment]
➤ Stefan Vladut commented:

Andrei Bodea this is an older issue and if I remember right the cards section was ignored by voice over so it was not saying anything, if it reads the info when you tap the card info then it should be ok

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v122 (37590) with iPhone 15 Pro (17.1.2).
